react-samples/eslint.config.js

```diff
@@ -76,12 +76,6 @@ export default [
           ignoreRestSiblings: true,
         },
       ],
-      '@typescript-eslint/no-floating-promises': [
-        'error',
-        {
-          ignoreIIFE: true,
-        },
-      ],
```

Comment:
I removed this rule because it complains about the floating promise in:

```ts
useEffect(() => {
  getJWT().then((jwt) => setJwt(jwt));
}, []);
```